lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -39,54 +38,58 @@ class ProgressTabView(RetrieveAPIView):
-            requirements: object containing the following fields
-                display_name: (str) the name of the requirement that should be displayed
-                namespace: (str) the type that the requirement is
-                min_grade: (float) the percentage formatted minimum grade needed for this requirement
-                status: (str) the status of the requirement
-                status_date: (str) the date the status was set
-        credit_support_url: (str) the url to the support docs for purchasing a credit
-        courseware_summary: List of serialized Chapters. each Chapter has the following fields:
+        completion_summary: Object containing unit completion counts with the following fields:
+            complete_count: (float) number of complete units
+            incomplete_count: (float) number of incomplete units
+            locked_count: (float) number of units where contains_gated_content is True
+        course_grade: Object containing the following fields:
+            percent: (float) the user's total graded percent in the course
+            is_passing: (bool) whether the user's grade is above the passing grade cutoff
+        courseware_point_summary: List of serialized Chapters. Each Chapter has the following fields:
             display_name: (str) a str of what the name of the Chapter is for displaying on the site
             subsections: List of serialized Subsections, each has the following fields:
+                assignment_type: (str) the format, if any, of the Subsection (Homework, Exam, etc)
                 display_name: (str) a str of what the name of the Subsection is for displaying on the site
-                due: (str) a DateTime string for when the Subsection is due
-                format: (str) the format, if any, of the Subsection (Homework, Exam, etc)
-                graded: (bool) whether or not the Subsection is graded
-                graded_total: an object containing the following fields
-                    earned: (float) the amount of points the user earned
-                    possible: (float) the amount of points the user could have earned
-                percent_graded: (float) the percentage of the points the user received for the subsection
+                total_points: an object containing the following fields
+                    num_earned: (float) the amount of points the user has earned for the given subsection
+                    num_possible: (float) the total amount of points possible for the given subsection
+                is_graded_assignment: (bool) whether or not the Subsection is a graded assignment
+                percent_graded: (float) the percentage of total points the user has received a grade for in a given subsection
                 show_correctness: (str) a str representing whether to show the problem/practice scores based on due date
+                                  ('always', 'never', 'past_due')
```

Comment:
Is this the full enumeration of options? Might be good to point to the piece of code these are defined in case they ever change.